docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3934 @@
+* Adds the ability to remove empty headers when there is no x-axis in *Lens* {kibana-pull}153420[#153420]
+* Adds improvements to the Metric formatter to support bit format in *Lens* {kibana-pull}153389[#153389]
+* Adds the ability to prevent default behaviour from action callback in *Lens* {kibana-pull}152842[#152842]
+* Adds Random Sampling to *Lens* {kibana-pull}151749[#151749]
+
+Machine Learning::
+* DFA creation wizard: add ability to add time field to result data view {kibana-pull}155669[#155669]
+* Display info when no datafeed preview results are found {kibana-pull}155650[#155650]
+* Adding ignore unavailable indices option to anomaly detection job wizards {kibana-pull}155527[#155527]
+* Support multiple model deployments {kibana-pull}155375[#155375]
+* Uses two weeks before now for default start time in job start date picker {kibana-pull}155312[#155312]
+* AIOps: Adds filter action for the Change point detection results {kibana-pull}155256[#155256]
+* Adds search links for AIOps Labs pages {kibana-pull}155202[#155202]
+* AIOps: Adds field stats for metric and split fields {kibana-pull}155177[#155177]
+* AIOps: Link from Explain Log Rate Spikes to Log Pattern Analysis {kibana-pull}155121[#155121]
+* Explain Log Rate Spikes: adds popover to analysis table for viewing other field values {kibana-pull}154689[#154689]
+* Explain Log Rate Spikes: Makes use of random sampling for overall histogram chart {kibana-pull}154520[#154520]
+* Explain Log Rate Spikes: Adds table action to copy filter to clipboard {kibana-pull}154311[#154311]
+* Data Frame Analytics: support custom urls in jobs {kibana-pull}154287[#154287]
+* Change point detection: support for multiple metric and split fields {kibana-pull}154237[#154237]
+* Enhances support for counter fields in data visualizer / field statistics {kibana-pull}153893[#153893]
+* Explain Log Rate Spikes: Support to filter fields from grouping {kibana-pull}153864[#153864]
+* Custom sorting by message level on Notifications page {kibana-pull}153462[#153462]
+* Log pattern analysis in Discover {kibana-pull}153449[#153449]
+* Explain Log Rate Spikes: Improves grouping using the `include` option of the `frequent_item_sets` agg {kibana-pull}153091[#153091]
+* Data Frame Analytics exploration: adds actions column with link to discover {kibana-pull}151482[#151482]
+* Allows row expansion for blocked anomaly detection jobs {kibana-pull}151351[#151351]
+* Enhances job and datafeed config editors in the Advanced anomaly detection job wizard to provide suggestions and documentation {kibana-pull}146968[#146968]
+
+Management::
+* Adds timezone support for Transforms date histogram pivot configuration {kibana-pull}155535[#155535]
+* Adds more system indices for storing internal data {kibana-pull}154888[#154888]
+* Adds option to Reauthorize transform in Management page {kibana-pull}154736[#154736]
+* Adds secondary authorization header to Transforms in Fleet {kibana-pull}154665[#154665]
```

Comment:
Maybe this should be grouped under `Machine Learning` or `Fleet`, I'm not sure. @qn895 WDYT?

Reply:
@peteharverson I think `Adds option to Reauthorize transform in Management page` should be under Machine Learning, and `Adds user's secondary authorization to Transforms in Fleet` under Fleet. Wdyt?

Reply:
`Adds option to Reauthorize transform in Management page` needs to go into the Management section (and not ML), as the Transforms feature is inside the Management page.

`Adds user's secondary authorization to Transforms in Fleet` should go under Fleet.